src/patternfly/components/Accordion/accordion.scss

```diff
@@ -4,7 +4,6 @@
 .pf-c-accordion {
   // accordion
   --pf-c-accordion--BackgroundColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-accordion--BorderWidth: var(--pf-global--BorderWidth--sm);
```

Comment:
are you making this PR a breaking change? or is this considered a bug?

Reply:
This var isn't used anywhere in the component, so it's safe to remove.

Reply:
I can add it back and create a follow up issue to remove it in a breaking change release if you can think of a use case where removing it would break someone's design.

Reply:
I'm not sure if I can think of a use case of it breaking someones design, but to be safe I think we've been following that method right? @mcoker

Reply:
I think we did leave it one place before in a similar scenario. I think the use case would be if someone had written their own CSS that used `--pf-c-accordion--BorderWidth`, removing the var would break what they're doing. I'll leave it and leave a comment to remove it at a breaking change release just to be sure.